script.js

```diff
@@ -0,0 +1,116 @@
+class Collection {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+let book = [];
+
+// Book displayed on the page
+
+function init() {
+  for (let i = 0; i < book.length; i += 1) {
```

Comment:
- [OPTIONAL] 

- [ ] Consider changing the functions in your code to the ES6 [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) as it is a good practice and makes your code shorter and precise.

Reply:
Thank you for your review! We will use the arrow function :slightly_smiling_face: